server/src/main/java/io/druid/server/AsyncQueryForwardingServlet.java

```diff
@@ -321,7 +321,7 @@ public void onComplete(Result result)
                         "query/time",
                         requestTime,
                         "success",
-                        result.isSucceeded()
+                        result.isSucceeded() && result.getResponse().getStatus() == 200
```

Comment:
why this changed? also can you use some existing static variable from http libs instead of 200?